packages/generator-prismic-nuxt/generators/app/index.ts

```diff
@@ -59,7 +59,16 @@ export default class Nuxt extends PrismicGenerator {
     const nuxtPrompts = NuxtPrompts.map(p => {
       if (p.name === 'name') return {...p, default: this.domain}
       // do the same for github user name
-      if (p.name === 'gitUsername') return {...p, default: this.user.github.username}
+      if (p.name === 'gitUsername') return {
+        ...p,
+        default: () => {
+          try {
+            return this.user.github.username()
```

Comment:
username triggers an error if doesn't exist? that weird, I'd expect a null and then we can fallback on something else.
One more question, why `this.user.github.username` becomes `this.user.github.username()` ?
wasn't it a function before? I don't get this part.

Reply:
if default is a function inquirer will call it, `this.user.github.username` is a function, 
last line here https://yeoman.github.io/generator/actions_user.js.html
The docs seem wrong as it's not promise, I tested it and it will throw if the github user name is not found

Reply:
then `this.user.git.name() || this.user.git.email()` can break the same way and it won't be caught